src/web/modules/custom/workbc_cdq_career_match/js/workbc-cdq-career-match.js

```diff
@@ -303,4 +303,28 @@
     }
   };
 
+
+  Drupal.behaviors.CareerQuizzes = {
+    attach: function (context, settings) {
+
+      $("#block-workbc-cdq-content > form > div.form-wrapper").each(
+        function () {
```

Comment:
Just to confirm: The logic of injecting the `error` class is done in the core webform module? So all we need to do here is style it differently.

Reply:
yes